src/transformers/models/align/processing_align.py

```diff
@@ -31,7 +31,7 @@ class AlignProcessor(ProcessorMixin):
     Args:
         image_processor ([`EfficientNetImageProcessor`]):
             The image processor is a required input.
-        tokenizer ([`BERTTokenizer`, `BertTokenizerFast`]):
+        tokenizer ([`BertTokenizerFast`]):
```

Comment:
Typo should be fixed but it can be a slow or fast one.

Reply:
I will just fix the typo then